docs/index.md

```diff
@@ -135,23 +165,29 @@ custom: {
 
 <div class="Row">
     <div class="Fl(start) W(60%) Fl(n)--xs W(a)--xs">
-        <p>What do you prefer? Floats? Flexbox? Inline-block? CSS table? <a href="{% link tutorials/layout.md %}#layouts">Atomizer supports it all</a>, and you can use <a href="{% link tutorials/layout.md %}#widths">any measurement style you want</a> (fraction, percentage, em, rem, px, etc.)</p>
+        <p>What do you prefer? Floats? Flexbox? Inline-block? CSS table? <a href="{% link tutorials/layout.md %}#layouts">Atomizer supports it all</a>.</p>
```

Comment:
Assume the rest of these changes were intentional...

Reply:
Ya since we got rid of the Widths section on the Layout page